docs/src/modules/java-protobuf/pages/actions.adoc

```diff
@@ -360,6 +360,20 @@ action returning and the `addItem` call from the action would lead to more items
 
 Such validation that depends on state can only be done safely while handling the command inside the entity.
 
+== Actions as Life-cycle Hooks
+
+An Action's gRPC method can be triggered automatically when some predefined service life-cycle event happens, serving as a custom hook. For such use, the method needs be annotated with a specific flag as shown below and its must be of type `google.protobuf.Empty`.
+
+[source,proto]
+----
+include::example$java-protobuf-doc-snippets/src/main/proto/com/example/on_startup.proto[tag=hook]
+----
+<1> only methods belonging to an Action can be configured as a hook;
+<2> method must receive `google.protobuf.Empty` as input type;
+<3> select which event triggers this hook (in this case, this is once instance startup is concluded);
```

Comment:
We only have STARTUP, so it can be misleading to say that we can select the event.